tools/cmake/util.cmake

```diff
@@ -1,19 +1,3 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-# PREFIX - TARGET or similar (can be empty string)
```

Comment:
Maybe delete this file instead of leaving it empty. You will also have to look for places it might have been included.